bento_beacon/utils/katsu_utils.py

```diff
@@ -236,10 +256,20 @@ def katsu_datasets(id=None):
 
 def phenopackets_for_ids(ids):
     # retrieve from katsu search
-
     payload = {
         "data_type": "phenopacket",
         "query": ["#in", ["#resolve", "subject", "id"], ["#list", *ids]]
     }
     endpoint = current_app.config["KATSU_SEARCH_ENDPOINT"]
     return katsu_network_call(payload, endpoint)
+
+
+def search_summary_statistics(ids):
+    endpoint = "/api/search_overview"
```

Comment:
some endpoints are in config and some aren't - they don't necessarily need to be configurable, but it would be good to have a consistent place where katsu endpoint paths are stored

Reply:
yeah.. I got tired of having a long list of configurable endpoints that will never change, so started baking them into the code. I can throw them back into config if it looks more consistent.